en_US/getting-started/new-features.md

```diff
@@ -101,11 +107,17 @@ In EMQX 5.1, we have redesigned the EMQX Dashboard, enhancing the visual experie
 - Visualized data flows: Using [Flows](../dashboard/flows.md) users can clearly see how data flows from devices or clients through the rule engine
 - Configuration updates during runtime: Hot update that takes effect immediately
 
+## Overload protection, Limiter and Buffer queue for Bridges
+
+New **Limiter** feature delivers a more accurate and layered rate control and ensures that the system operates under the expected workload, because it supports limiting client behavior at the client, listener, or node levels. The combination of Overload protection and Limiter prevents the clients from becoming too busy or receiving excessive request traffic and ensures stable system operation.
+
+We also added a generic buffer queue for all bridges which allows to buffer messages generated under stressful conditions. It can be configured to store messages in memory or disk cache when external resources are unavailable, for example, due to network fluctuations or service downtime. Messages will be sent after the service is restored. However, requests may expire in the buffer – this is a big difference comparing with v4. If the buffered data exceeds the limit, it will be discarded following the First In First Out (FIFO) rule.
```

Comment:
```suggestion
A generic buffer queue is also added for all bridges, allowing buffer messages generated under stressful conditions. This buffer can be configured to store messages in either memory or disk cache when external resources are unavailable, such as during network fluctuations or service downtime. Buffered messages will be sent once the service is restored. However, requests in the buffer may expire, which is a big difference compared to version 4. If the amount of buffered data exceeds the limit, it will be discarded following the First In First Out (FIFO) rule.
```